src/app/park-form/park-form.component.html

```diff
@@ -1,111 +1,149 @@
                             </div>
                         </label>
                     </div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-12">
                     <div class="form-group custom-control custom-switch">
-                        <input type="checkbox" class="custom-control-input" id="visibility"
-                            formControlName="visibility">
-                        <label class="custom-control-label d-inline-flex" for="visibility">
+                        <input type="checkbox"
+                            class="custom-control-input"
+                            id="visible"
+                            formControlName="visible">
+                        <label class="custom-control-label d-inline-flex"
+                            for="visible">
                             Park&nbsp;Visibility:
-                            <div *ngIf="parkForm.get('visibility').value" class="mx-1 text-primary">Park is visible
-                                to
-                                the
-                                public.
+                            <div *ngIf="parkForm.get('visible').value"
+                                class="mx-1 text-primary">
+                                Park is visible to the public.
+                            </div>
+                            <div *ngIf="!parkForm.get('visible').value"
+                                class="mx-1 text-secondary">
+                                Park is not visible to the public.
                             </div>
-                            <div *ngIf="!parkForm.get('visibility').value" class="mx-1 text-secondary">Park is not
-                                visible to
-                                the
-                                public.</div>
                         </label>
                     </div>
-                    <div class="flex-fill"></div>
-                <div class="row">
-                    <div class="form-group col-md-5">
-                        <label for="name">Park Name *</label>
-                        <input type="text" class='form-control' id="name" formControlName="name"
-                            placeholder="Enter park name" required>
-                    </div>
-                    <div class="form-group col-md-5">
-                        <label for="link">Link to Park Site</label>
-                        <div class="input-group">
-                            <input id="link" formControlName="link" class="form-control" type="text"
-                                placeholder="Paste link here">
-                            <div class="input-group-append">
-                                <button (click)="testLink()" [disabled]="!parkForm.get('link').value" class="btn btn-primary" type="button">Test Link</button>
-                            </div>
+                </div>
+            </div>
+
+            <div class="row">
```

Comment:
disable the park name field when editing park - changing name creates new park

Reply:
068e8a9f780f2d0e3f81771cb8ee53f01755fa68